build/test_bilder.py

```
@@ -20,6 +21,63 @@ def png_masse(pfad):
+        return a if pa <= pb and pa <= pc else (b if pb <= pc else c)
+
+    schritt = breite * 4                        # RGBA
+    vorige, alpha = bytearray(schritt), []
+    for y in range(hoehe):
+        anfang = y * (schritt + 1)
+        filt, zeile = roh_zeilen[anfang], bytearray(roh_zeilen[anfang + 1:anfang + 1 + schritt])
+        for i in range(schritt):
+            a = zeile[i - 4] if i >= 4 else 0
+            b = vorige[i]
+            c = vorige[i - 4] if i >= 4 else 0
+            if filt == 1:   zeile[i] = (zeile[i] + a) & 0xFF
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)